### PR TITLE
[15.0][MIG] stock_barcodes_gs1_expiry: Migration to v15

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa

Depends on:

- [x] https://github.com/OCA/stock-logistics-barcode/pull/495
- [x] https://github.com/OCA/stock-logistics-barcode/pull/543

### CODE REVIEW

**Comment by sergio-teruel** [comment]
Ready to be merged!

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Comment by pedrobaeza** [comment]
@sergio-teruel please rebase and include `odoo_test_helper` in test-requirements.txt.

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# generated from manifests external_dependencies
```
Comment (pedrobaeza) [inline test-requirements.txt]: This text is not really correct

**Comment by sergio-teruel** [comment]
Changes done1!

**Pull request merged**